src/components/specific/users/project-users-manager/ProjectUsersManager.vue

```diff
@@ -36,6 +36,7 @@
                   v-if="showInvitationForm"
                   key="invitation-form"
                   :project="project"
+                  :displayedUsers="displayedUsers"
```

Comment:
I think you have to pass the **full** list of users here, not only **displayed** users.
Indeed, if you enter a search text ti filter user list you might have users that are not displayed but are already in the project. ;-)

I suggest to rename the props from `displayedUsers` to `users` and pass all the users in it.
```html
<InvitationForm
   v-if="showInvitationForm"
   key="invitation-form"
   :project="project"
   :users="users"
   @close="closeInvitationForm"
   @success="onInvitationSuccess"
/>
```